src/assets/base/base.scss

```diff
@@ -1,18 +1,257 @@
+  align-items: center;
+  justify-content: center;
+}
+
+.BC-Gov-SecondaryButton {
+    background: none;
+    border-radius: 4px;
+    border: 2px solid #003366;
+    padding: 10px 30px;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 18px;
+    Font-family: ‘BCSans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
+    font-weight: 700;
+    letter-spacing: 1px;
+    cursor: pointer;
+    color: #003366;
+    }
+.BC-Gov-SecondaryButton:hover {
+    opacity: 0.80;
+    text-decoration: underline;
+    background-color: #003366;
+    color: #FFFFFF;
+    }
+:focus {
+    outline-offset: 1px;
+    outline: 4px solid #3B99FC;
+    }
+.BC-Gov-SecondaryButton:active {
+    opacity: 1;
+    }
+
+.BC-Gov-SecondaryButton-Dark {
+    background: none;
+    border-radius: 4px;
+    border: 2px solid #fff;
+    padding: 10px 30px;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 18px;
+    Font-family: ‘Noto Sans’, Verdana, Arial, sans-serif;
+    font-weight: 700;
+    letter-spacing: 1px;
+    cursor: pointer;
+    color: #fff;
+    }
+
+.BC-Gov-SecondaryButton-Dark:hover {
+    text-decoration: underline;
+    background-color: #fff;
+    color: #313132;
+    }
+
+:focus {
+    outline-offset: 1px;
+    outline: 4px solid #3B99FC;
+    }
+
+.BC-Gov-SecondaryButton-Dark:active {
+    background-color: #f2f2f2;
+    color: #313132;
+    }
+
+.background-colour {
+    background-color: #003366;
+    height: 100px;
+    width: 300px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    }
+
+.BC-Gov-PrimaryButton-disabled {
+    background-color: #003366;
+    opacity: 0.3;
+    border: none;
+    border-radius: 4px;
+    color: white;
+    padding: 12px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 18px;
+    font-family: ‘BCSans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
+    font-weight: 700;
+    letter-spacing: 1px;
+    cursor: not-allowed;
+}
+
+.BC-Gov-SecondaryButton-disabled {
+    background-color: white;
+    opacity: 0.3;
+    border: 2px solid #003366;
+    border-radius: 4px;
+    color: #003366;
+    padding: 10px 30px;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 18px;
+    font-family: ‘Noto Sans’, Verdana, Arial, sans-serif;
+    font-weight: 700;
+    letter-spacing: 1px;
+    cursor: not-allowed;
+}
+
+// cards
+
+.card-header {
+    background-color: $card-header-bg;
+    color: $card-header-color;
+    font-weight: bold;
+    h1 {
+        font-size: 28px;
+        margin-bottom: 0px;
+    }
+}
+
+  .park-select-header {
+    background-color: $card-header-bg;
+    color: $card-header-color;
+    font-weight: bold;
+    h2 {
+        font-size: 28px;
+        margin-bottom: 0px;
+    }
+}
+
+@media (min-width: 576px) {
+    .main-content {
+        margin-bottom: 240px;
+    }
+
+    .header-primary {
+        margin-top: 60px;
```

Comment:
hard-coding spacing values may cause issues with adaptive display - use bootstrap spacing classes in angular template files instead. (typ)